_overviews/scala3-macros/tutorial/inline.md

```diff
@@ -86,8 +86,8 @@ val level   = logLevel
 def message = getMessage()
 
 println(s"[$level]Computing $message")
-val res = computeSomthing()
-println(s"[$level]Result of $tag: $res")
+val res = computeSomething()
+println(s"[$level]Result of $message: $res")
```

Comment:
'tag' does not appear anywhere else and if I understand correctly the by-name parameter should be evaluated twice (i.e. the call to the 'message' def)